docs/api/geo_datasets.csv

```diff
@@ -12,6 +12,7 @@ Dataset,Type,Source,Size (px),Resolution (m)
 `GBIF`_,Points,Citizen Scientists,-,-
 `GlobBiomass`_,Masks,Landsat,"45,000x45,000",100
 `iNaturalist`_,Points,Citizen Scientists,-,-
+`L7 Irish`_,"Imagery, Masks",Landsat,"8,400x7,500","15, 30"
 `LandCover.ai Geo`_,"Imagery, Masks",Aerial,"4,200--9,500",0.25--0.5
 `Landsat`_,Imagery,Landsat,"8,900x8,900",30
 `L8 Biome`_,"Imagery, Masks",Landsat,"8,900x8,900","15, 30"
```

Comment:
Can you move L8 Biome underneath L7 Irish? I think it was in the wrong location.